zfec/filefec.py

```diff
@@ -277,14 +279,14 @@ def decode_from_files(outf, infiles, verbose=False):
             byteswritten += len(resultdata)
             if verbose:
                 if ((byteswritten - len(resultdata)) / (10*MILLION_BYTES)) != (byteswritten / (10*MILLION_BYTES)):
-                    print str(byteswritten / MILLION_BYTES) + " MB ...",
+                    print(str(byteswritten / MILLION_BYTES) + " MB ...", end=" ")
         else:
             if padlen > 0:
                 outf.truncate(byteswritten - padlen)
             return # Done.
     if verbose:
-        print
-        print "Done!"
+        print()
```

Comment:
I think this is done to get "Done!" printed on new line and can be replaced with single statement like below
```python
print("\nDone!")
```